tasks/esi_node_network_attach_task.yaml

```diff
@@ -0,0 +1,31 @@
+{
+    "version": 2,
+    "title": "A cropped version of a bigger task.",
+    "description": "Auto-generated task from a single workload",
```

Comment:
The title/description should probably be updated

Reply:
Corrected.